source/_integrations/mqtt.markdown

```diff
@@ -197,7 +197,7 @@ To change the settings, follow these steps:
 
 1. Go to **{% my integrations title="Settings > Devices & services" %}**.
 2. Select the MQTT integration.
-3. Select **Configure**, then **Re-configure MQTT**.
+3. Select **Add**, then **Re-configure MQTT**.
```

Comment:
I think this is also related to an already existing config entry, those related to the options flow?